packages/kit/src/exports/vite/dev/index.js

```diff
@@ -419,8 +442,36 @@ export async function dev(vite, vite_config, svelte_config) {
 	});
 
 	const env = loadEnv(vite_config.mode, svelte_config.kit.env.dir, '');
+	// Update the `env` used in the `sveltekit_environment_context` virtual module.
+	environment_context.env = env;
 	const emulator = await svelte_config.kit.adapter?.emulate?.();
 
+	/**
+	 * The environment that was provided to `kit.environments.ssr` in the Svelte config.
+	 * @type { ((import('vite').DevEnvironment & { api?: { getHandler: (opts: { entrypoint: string }) => Promise<(req: Request) => Promise<Response>> }})) | undefined }
+	 */
+	const devEnv = vite.environments[SSR_ENVIRONMENT_NAME];
```

Comment:
I know our naming scheme is unusual. sorry...
```suggestion
	const dev_env = vite.environments[SSR_ENVIRONMENT_NAME];
```